### ISSUE TITLE
Reformat code eats whitespace after a comma in footnotes

### ISSUE THREAD

**Issue by slonopotamus** [asciidoctor/asciidoctor-intellij-plugin#692]
## Steps to reproduce

1. Create AsciiDoc document with the following contents:

```
sometextfootnote:[one, two, three]
```

2. Apply Reformat

Expected: document is not modified

Actual: whitespace in footnote text is gone

```
sometextfootnote:[one,two,three]
```

## Environment

Plugin Version: 0.32.26

IntelliJ Details:
```
RubyMine 2020.2.3
Build #RM-202.7660.39, built on October 8, 2020
Runtime version: 1.8.0_275-b01 amd64
VM: OpenJDK 64-Bit Server VM by IcedTea
Linux 5.4.80-gentoo-r1
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 16
Non-Bundled Plugins: Batch Scripts Support, org.asciidoctor.intellij.asciidoc
Current Desktop: KDE
```

**Comment by slonopotamus** [comment]
Correction: this only affects whitespace *after a comma*.

**Comment by ahus1** [comment]
Thanks for reporting this. I'll need to update the lexer for this; with footnotes a comma doesn't separate attributes. For footnotes it should be treated as normal text.

**Comment by ahus1** [comment]
I've published pre-release 0.32.27 that should fix the problematic re-formatting. I needed to re-factor some existing code. All tests are green, but I might have missed corner cases. Please give it a try. 

The pre-release of the plugin is available from [GitHub releases](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/installing-a-preview-release.html) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).

**Comment by ahus1** [comment]
Pre-release 0.32.28 adds spell checking for footnote content. I'm closing this issue as no negative feedback came up so far.

Please comment here and I'll re-open the issue.
